gatsby-node.js

```diff
@@ -2,25 +2,22 @@ const path = require(`path`)
 const { createFilePath } = require(`gatsby-source-filesystem`)
 const contributorsList = require('./contributors.json')
 let stats
-if (process.env.ENV_TYPE === 'mock') {
+if (true) {
```

Comment:
Let's revert the changes in this file

Reply:
yes, doing it rn.

Reply:
@Shubham-Agarwal16 Changes for the file reverted.